### PR TITLE
Replace 404 redirect for incorrect query

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

**Comment by admiral96** [comment]
Travis-CI build failed once. Re-correcting...

File: app/views.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def print_user_input(query):
     if '=' in query:
         query_container, query_value = query.split('=')
         return 'Your query is {} which is equal to {}'.format(query_container, query_value)
-    return page_not_found(404)
+    return "You've entered an incorrect query. Please check and try again. Input : {}".format(query)
```
Comment (mknecht) [inline app/views.py]: nice error message! can you be slightly more specific, though? what did the client not do?

what status code is this returning at the moment? the best thing is to add a test that makes sure that a 400 is being returned.

**Comment by admiral96** [comment]
@mknecht Please review

**Comment by mknecht** [comment]
nice! one test away from merging! :)

**Pull request merged**